README.md

```diff
@@ -153,8 +154,13 @@ The interfaces mimic regular Background Sync, but substitute `navigator.sync` wi
 ## Power and Network conditions
 The original explainer allowed the website to specify power and network conditions under which the
 periodic task should be run. This proposal leaves those decisions to the browser implementation,
-because we believe the browser is better positioned to make those choices for the user. This also
-makes for a simpler interface for developers.
+because we believe the browser is better positioned to make those choices for the user.
+This also makes for a simpler interface for developers.
+
+It is recommended however, that the browser respect user indications such as Data Saver and Battery
+Saver modes and skip sending periodicsync events when the user has turned these on. This would also
```

Comment:
Data Saver and Battery Saver, while descriptive, isn't the cross-platform terminology to use due to the capitalization. You could rephrase as:

> When the device is in either battery or data saving mode, the `periodicsync` event SHOULD not be fired.

That also adds some rfc2119ness :).